e2e-tests/charge-cycles-on-reject.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+set -Eexuo pipefail
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+source "${SCRIPT_DIR}/utils.sh"
+
+# Run dfx stop if we run into errors.
+trap "dfx stop" EXIT SIGINT
+
+dfx start --background --clean
+
+# Deploy the canister that returns the blocks for harge-cycles-on-reject.
+dfx deploy e2e-scenario-3
+
+# Deploy the bitcoin canister, setting the blocks_source to be the source above.
+dfx deploy bitcoin --argument "(record {
+  stability_threshold = 2;
+  network = variant { regtest };
+  blocks_source = principal \"$(dfx canister id e2e-scenario-3)\";
+  syncing = variant { enabled };
```

Comment:
nit: We don't need to be syncing blocks in this test.

```suggestion
  syncing = variant { disabled };
```